go.mod

```diff
@@ -1,6 +1,6 @@
 module helm.sh/helm/v3
 
-go 1.17
+go 1.18
```

Comment:
Sorry, about this. I wanted to use the sets [package](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets) which uses genereics.